docs/developer/issue-triage-process.md

```diff
@@ -0,0 +1,90 @@
+- Repeat until all issues in the **Inbox** column are triaged.
+
+Besides this manual triage process, there are two automatic process lead by the bots configured in GitHub:
++ Automatically move new and reopened issues to the **Inbox** column to start the triage process (due to eventual project-bot outages, once a month we should manually search for issues with "no:project" assigned).
++ Automatically **dependabot** and **Snyk** are labelling PRs depending on the target language.
++ Automatically **stalebot** is checking inactive issues to label them as `stale`. An issue becomes stale after 60 days of inactivity and the bot will close it after 7 days of inactivity for stale issues. To be considered:
+  * Issues labeled as `priority/unbreak-now` → `priority/high` → `priority/medium` will never be labeled as `stale`.
+  * Only issues labeled as `priority/low` or `awaiting-more-evidence` could be considered stale (and those without any priority label).
+  * The label to use when marking an issue as stale is `stale`.
+
+## Labels
+
+### Component
+
+component/| | | | | |
+----------|-|-|-|-|-|
+`apprepository-controller`|`asset-syncer`|`assetsvc`|`auth`|`chart`|`ci`
+`dashboard`|`docs`|`hub`|`kubeops`|`project`|`pinniped-proxy`
+
+### Kind
+
+kind/| | | |
+-----|-|-|-|
+`bug`|`feature`|`question`|`refactor`
+
+### Size
+
+size/| |
+-----|-|
+`XS`|A task that can be done by a person in less than 1 full day
+`S`|A story that can be done by a person in 1-3 days, with no uncertainty
+`M`|A story that can be done by a person in 4-7 days, possibly with some uncertainty
+`L`|A story that requires investigation and possibly will take a person a full 2-week iteration
+`XL`|A story too big or with too many unknowns. Needs investigation and split into several ones
+
+### Priority
+
+priority/| | | |
+--------|-|-|-|
+`unbreak-now`|`high`|`medium`|`low`
+
+
+**Contribution labels**:
+
+`awaiting-more-evidence` - information requested to the reporter.
+
+`help-wanted` - the maintainer team wants help on an issue or pull request.
+
+`good-first-issue` - good first issues to start contributing to Kubeapps.
+
+Automatic labels| | | | | |
+----------------|-|-|-|-|-|
+`dependencies`|`go`|`javascript`|`rust`|`security`|`stale`
+
+
+## Kubeapps milestones
+Next step in the triage process to help the community understand the project roadmap is to define milestones and associate issues to them.
+According to the Kubeapps team practices, milestones will be defined based on quarters and years according to the following pattern: **YYYY-QN** (i.e. 2021-Q1 - 2021-Q2 - etc), setting the due date for each milestone.
+
+## Kubeapps iteration planning guidelines
+
+1. Review issues in column **Inbox** (untriaged issues, awaiting-more-evidence, triaged issues labeled as low and medium priority) and move them to the column according to the triage process. Issues labeled as `awaiting-more-evidence` must be checked if updated to be triaged.
```

Comment:
Perhaps it's not relevant here as it's not part of triaging, but we'd normally go through the current in-progress work to get an update so that we also have a better idea of how much we can commit to for the next iteration.